src/apis/user/user.resolver.ts

```diff
@@ -17,13 +17,26 @@ export class UserResolver {
 
   //닉네임 체크
   @Mutation(() => String)
-  async checkNickName(
+  async checkNickname(
     @Args('nickname') nickname: string, //
   ) {
-    const findNickName = await this.userService.findOne({ nickname });
-    if (findNickName) {
+    const findNickname = await this.userService.findOne({ nickname });
+    if (findNickname) {
       return false;
-    } else return true;
+    }
+    return true;
+  }
+
+  //이메일 체크
+  @Mutation(() => String)
```

Comment:
```suggestion
  @Mutation(() => Boolean)
```
해당 mutation 은 boolean 을 반환하니 Boolean 타입으로 돼야할거같습니다.
위에 checkNickname 도 동일합니다.

Reply:
앗 그러네요 반영하겠습니다!